src/NServiceBus.Core/Transports/Msmq/MsmqConnectionStringBuilder.cs

```diff
@@ -39,6 +39,11 @@ public MsmqSettings RetrieveSettings()
                 settings.TimeToReachQueue = TimeSpan.Parse((string) this["timeToReachQueue"]);
             }
 
+            if (ContainsKey("messageEnumeratorTimeout"))
```

Comment:
V7 doesn’t support this via the connectionstring since we have decided to move away from custom connection strings. Should we make v6 behave in the same way or are we ok with them being different?

Reply:
@andreasohlund I would just leave it in the connection string

Reply:
The we need to log obsolete warnings and add that to the upgrade guide

Reply:
We also don’t have a public api for v7 so what would you say in the upgrade guide?